src/components/ReleaseVersionMarkers.svelte

```diff
@@ -10,7 +10,7 @@
   const xScale = getContext('xScale');
 
   let markers = [];
-  switch ($store.product) {
+  switch ($store.searchProduct) {
```

Comment:
Currently getting this error in the console for all FOG metrics:

<img width="421" alt="CleanShot 2022-04-14 at 15 29 32@2x" src="https://user-images.githubusercontent.com/28797553/163462506-5dcace12-0f0c-48df-ae84-f4ac06507f7d.png">

We should use `searchProduct` instead of `product` to be inclusive of FOG.